src/operations/buy.ts

```diff
@@ -56,9 +68,45 @@ export const buyOperationHandler: OperationHandler<BuyOperation> = {
     scope: OperationScope
   ): Promise<BuyOutput> {
     const builder = await buyBuilder(metaplex, operation.input, scope);
+
+    const { blockhash, lastValidBlockHeight } = await metaplex.connection.getLatestBlockhash();
+
+    const lookupTableAccount = await metaplex.connection
+      .getAddressLookupTable(operation.input.addressLookupTable)
+      .then((res) => res.value);
+
+    const messageV0 = new TransactionMessage({
+      payerKey: builder.getFeePayer()!.publicKey,
+      recentBlockhash: blockhash,
+      instructions: builder.getInstructions(),
+    }).compileToV0Message([lookupTableAccount!]);
+
+    const transactionV0 = new VersionedTransaction(messageV0);
+    //transactionV0.sign(builder.getSigners()); // Does not work as Metaplex sdk's Signer type is different.
+    const signedTx = await operation.input.signTransaction(transactionV0);
```

Comment:
Its a bummer to have to pass in the signTransaction function. What is the error when using builder.getSigners? Is it a type mismatch?

It be good OS contribution to fix up metaplex sdk to support transactionV0.

Reply:
Yes, its a type mismatch. Metaplex already has plans to add VersionedTransaction once majority of wallet adapters are using them - https://github.com/metaplex-foundation/js/pull/326

Reply:
Can we type cast then?

Reply:
We can, but not sure if it will work out or not.